FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/util/AvatarUtils.kt

```diff
@@ -0,0 +1,217 @@
+                    email = context.getString(R.string.persona_email_carole_poland)
+            ),
+            createAvatar(
+                    context.getString(R.string.persona_name_mona_kane),
+                    email = context.getString(R.string.persona_email_mona_kane)
+            )
+    )
+}
+
+fun createSmallAvatarList(context: Context?): ArrayList<IAvatar> {
+    val context = context ?: return arrayListOf()
+    return arrayListOf(
+            createAvatar(
+                    context.getString(R.string.persona_name_amanda_brady),
+                    imageDrawable = ContextCompat.getDrawable(context, R.drawable.avatar_amanda_brady),
+                    email = context.getString(R.string.persona_email_amanda_brady)
+            ),
+            createAvatar(
+                    context.getString(R.string.persona_name_amanda_brady),
+                    email = context.getString(R.string.persona_email_amanda_brady)
+            ),
+            createAvatar(
+                    context.getString(R.string.persona_subtitle_researcher),
+                    email = context.getString(R.string.persona_email_lydia_bauer)
+            )
+    )
+}
+
+fun createImageAvatarList(context: Context?): ArrayList<IAvatar> {
+    val context = context ?: return arrayListOf()
+    return arrayListOf(
+            createAvatar(
+                    context.getString(R.string.persona_name_amanda_brady),
+                    imageDrawable = ContextCompat.getDrawable(context, R.drawable.avatar_amanda_brady),
+                    email = context.getString(R.string.persona_email_amanda_brady)
+            ),
+            createAvatar(
+                    context.getString(R.string.persona_name_daisy_phillips),
+                    imageDrawable = ContextCompat.getDrawable(context, R.drawable.avatar_daisy_phillips),
+                    email = context.getString(R.string.persona_email_daisy_phillips)
+            ),
+            createAvatar(
+                    context.getString(R.string.persona_name_elvia_atkins),
+                    imageDrawable = ContextCompat.getDrawable(context, R.drawable.avatar_elvia_atkins),
+                    email = context.getString(R.string.persona_email_elvia_atkins)
+            ),
+            createAvatar(
+                    context.getString(R.string.persona_name_colin_ballinger),
+                    imageDrawable = ContextCompat.getDrawable(context, R.drawable.avatar_colin_ballinger),
+                    email = context.getString(R.string.persona_email_colin_ballinger)
+            ),
+            createAvatar(
+                    context.getString(R.string.persona_name_katri_ahokas),
+                    imageBitmap = BitmapFactory.decodeResource(context.resources, R.drawable.avatar_katri_ahokas),
+                    email = context.getString(R.string.persona_email_katri_ahokas)
+            )
+    )
+}
+fun createAvatar(
+        name: String,
```

Comment:
Why createAvatar ? can't we use Avatar constructor itself ?
Also is Name Mandatory ?

Reply:
In Library we do not have concrete class for Avatar, We just have an IAvatar interface which is consumed by demo
So, not needed